packages/blockchain-utils-linking/package.json

```diff
@@ -26,6 +26,8 @@
   },
   "dependencies": {
     "@stablelib/sha256": "^1.0.0",
+    "@taquito/taquito": "^9.2.0",
+    "@taquito/utils": "^9.2.0",
```

Comment:
Please come up with a way to implement this without these packages. We are trying to keep the `blockchain-utils-linking` package small so that it can be easily bundled in browsers!

https://bundlephobia.com/package/@taquito/taquito@9.2.0 is 385.9 kB 
https://bundlephobia.com/package/@taquito/utils@9.2.0 is 69.5 kB

Reply:
`@taquito/taquito` can probably just be a dev dependency since it is really only used for typing.

`@taqutio/utils` is required tho since `char2Bytes` is custom.

Reply:
Ok, nice

Reply:
The actual compiled JS should only be a few 100 bytes due to [tree shaking](https://webpack.js.org/guides/tree-shaking/), but configuring this is outside the scope of this PR.

Reply:
Well tree shaking doesn't always work and if any developer that uses the ceramic http-client misconfigures their tree shaking they will end up blaming us. Please you create a simple type in your code rather than importing this library!

Reply:
> @taquito/taquito can probably just be a dev dependency since it is really only used for typing.

Could we just copy the types file instead of needing to import the entire thing? Fairly straight forward to do